src/files2move2bin/Config.TRACS

```diff
@@ -77,14 +77,14 @@
 Depth = 300.0   # Double in micrometers
 
 # Width of the read-out strips
-Width = 25.0   # Double in micrometers
+Width = 500 #25.0   # Double in micrometers
```

Comment:
I'd rather not have commented values here, it's not gonna have any impact in the software but I think it would be nicer to look at and easier to understand. What do you think?

Also I have never gotten a pull request from someone else, so I'm commenting every small thing so I test how it works